### ISSUE TITLE
Question about the batchnorm while trainning

### ISSUE THREAD

**Issue by AXu0511** [charlesCXK/TorchSemiSeg#82]
Hi Xiaokang,
If I have only a GPU, should i change the SyncBatchNorm to nn.BatchNorm2d?
Looking forward to your reply.
![image](https://user-images.githubusercontent.com/86408707/220721621-b1f8f281-28a2-41ee-9c69-1330b1512a20.png)

**Comment by charlesCXK** [comment]
Hi, I think it is ok.
